Sources/IndicatorInfo.swift

```diff
@@ -26,21 +26,13 @@ import Foundation
 
 public struct IndicatorInfo {
     
-    public var title: String
+    public var title: String?
     public var image: UIImage?
     public var highlightedImage: UIImage?
     
-    public init(title: String) {
+    public init(title: String? = nil, image: UIImage? = nil, highlightedImage: UIImage? = nil) {
```

Comment:
yep, i completely agree but first we need to define what are we going to allow.

Using a custom cell with other stuff shouldn't need any restructuring, there's already a function `configure(cell: ButtonBarCellType, for indicatorInfo: IndicatorInfo)` you can override in order to configure any custom cell that you use (youtube example)